standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/RawStore.java

```diff
@@ -906,23 +908,22 @@ int getNumPartitionsByPs(String catName, String dbName, String tblName, List<Str
    */
   default List<Partition> getPartitionsByNames(String catName, String dbName, String tblName,
```

Comment:
Should we deprecate this method?

Reply:
There are many callers invoke this method directly, and many of them don't care about the exclude/include key or  schema skipping, I would keep it as it is for easy use.